packages/contracts/test/LibGuiseLevel.t.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { MudV2Test } from "@latticexyz/std-contracts/src/test/MudV2Test.t.sol";
+import { LibGuiseLevel } from "../src/guise/LibGuiseLevel.sol";
+import { LibExperience } from "../src/charstat/LibExperience.sol";
+import { Experience, ExperienceTableId } from "../src/codegen/Tables.sol";
+import { ActiveGuise, ActiveGuiseTableId } from "../src/codegen/Tables.sol";
+import { GuisePrototype, GuisePrototypeTableId } from "../src/codegen/Tables.sol";
+import { PStat_length } from "../src/CustomTypes.sol";
+
+contract LibGuiseLevelTest is MudV2Test {
+  bytes32 internal targetEntity = keccak256("targetEntity");
+
+  function setUp() public virtual override {
+    super.setUp();
+    vm.startPrank(worldAddress);
+  }
+
+  function testGetAggregateLevel() public {
+    // Initialize exp and levelMul
+    LibExperience.initExp(targetEntity);
+    assertTrue(LibExperience.hasExp(targetEntity));
+
+    // Add exp
+    uint32[PStat_length] memory addExp = [uint32(1), 1, 1];
+    LibExperience.increaseExp(targetEntity, addExp);
+
+    //Add guise and set levelmul
+    uint256 guiseProtoEntity = ActiveGuise.get(targetEntity);
+    uint32[PStat_length] memory levelMul = [uint32(2), 2, 2];
+    GuisePrototype.set(bytes32(guiseProtoEntity), levelMul);
+
+    uint32 aggregateLevel = LibGuiseLevel.getAggregateLevel(bytes32(targetEntity));
+    assertEq(aggregateLevel, 1); // expected 1 = (2*1 + 2*1 + 2*1) / (2 + 2 + 2)
```

Comment:
for a one-off test I'd prefer more complicated values.
But better yet you should make a helper function instead and test several sets of edge cases.
And then you could fuzz it too